cli.js

```diff
@@ -16,11 +17,13 @@ const MochaFramework = require('./cli-commands/test/test-frameworks/mocha');
     const testCommand = new TestCommand();
     const deployCommand = new DeployCommand();
     const compileCommand = new CompileCommand();
+    const shapeCommand = new ShapeCommand();
```

Comment:
move it between other rows above so the final view will be ordered by lenght
...
.....
.......
..........